app/views/prison/visits/show.html.erb

```diff
@@ -21,6 +21,9 @@
       <% if (@visit.closed? && @visit.booked?) %>
         <small class="font-xsmall">(<%= t('.closed_visit') %>)</small>
       <% end %>
+      <% if @visit.cancelled? %>
+        <small class="font-xsmall">(<%= t('rejection_reason.'+@visit.cancellation_reason, scope: :shared) %>)</small>
```

Comment:
You could use string interpolation instead of `+`.

Where is the other place where the context is shared?